src/Analyzers/Core/Analyzers/RemoveUnusedMembers/AbstractRemoveUnusedMembersDiagnosticAnalyzer.cs

```diff
@@ -432,6 +437,11 @@ private void OnSymbolEnd(SymbolAnalysisContext symbolEndContext, bool hasUnsuppo
                                 continue;
                             }
 
+                            if (dynamicDependencyAttributeArguments.Any(signature => signature.Contains(member.Name)))
```

Comment:
yeah, that seems like a non-good check.  i think we'd have to generate the actual doc string for something and compare it to the attribute arg.

Reply:
@CyrusNajmabadi Yeah the current check is absolutely a bug. I was doing this quickly as an initial implementation that's not ready yet.